README.md

```diff
@@ -162,6 +165,7 @@ In order to use the API within `polkadot.js` you'll need to configure the follow
   "Restriction": {
     "_enum": {
       "None": "()",
+      "BooleanBinary": "BooleanBinaryRestriction",
```

Comment:
not sure about `BooleanBinary` as a name?

Reply:
`CombinedRestriction`?

Reply:
changed to `Combined` and `CombinedRestriction`